### ISSUE TITLE
Babel errors after upgrading to 6.0

### ISSUE THREAD

**Issue by guoyunhe** [JeffreyWay/laravel-mix#2795]
-   Laravel Mix Version: 6.0.10
-   Node Version (`node -v`): 14.15.4
-   NPM Version (`npm -v`): 6.14.8
-   OS: openSUSE Tumbleweed 20210118

### Description:

I was upgrading from 5.x to 6.0. Here is my webpack.mix.js:

```js
const mix = require('laravel-mix');

mix.ts('resources/ts/index.tsx', 'public/js/app.js').react().extract().sourceMaps();
```

But when I run `mix watch --hot`, I get some errors in babel:

```
> mix watch --hot

ERROR in ./node_modules/@babel/runtime/helpers/esm/objectWithoutProperties.js 1:0-74
Module not found: Error: Can't resolve './objectWithoutPropertiesLoose' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'objectWithoutPropertiesLoose.js'?
BREAKING CHANGE: The request './objectWithoutPropertiesLoose' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/slicedToArray.js 1:0-46
Module not found: Error: Can't resolve './arrayWithHoles' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'arrayWithHoles.js'?
BREAKING CHANGE: The request './arrayWithHoles' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/slicedToArray.js 2:0-58
Module not found: Error: Can't resolve './iterableToArrayLimit' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'iterableToArrayLimit.js'?
BREAKING CHANGE: The request './iterableToArrayLimit' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/slicedToArray.js 3:0-70
Module not found: Error: Can't resolve './unsupportedIterableToArray' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'unsupportedIterableToArray.js'?
BREAKING CHANGE: The request './unsupportedIterableToArray' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/slicedToArray.js 4:0-48
Module not found: Error: Can't resolve './nonIterableRest' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'nonIterableRest.js'?
BREAKING CHANGE: The request './nonIterableRest' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/toConsumableArray.js 1:0-52
Module not found: Error: Can't resolve './arrayWithoutHoles' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'arrayWithoutHoles.js'?
BREAKING CHANGE: The request './arrayWithoutHoles' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/toConsumableArray.js 2:0-48
Module not found: Error: Can't resolve './iterableToArray' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'iterableToArray.js'?
BREAKING CHANGE: The request './iterableToArray' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/toConsumableArray.js 3:0-70
Module not found: Error: Can't resolve './unsupportedIterableToArray' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'unsupportedIterableToArray.js'?
BREAKING CHANGE: The request './unsupportedIterableToArray' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

ERROR in ./node_modules/@babel/runtime/helpers/esm/toConsumableArray.js 4:0-52
Module not found: Error: Can't resolve './nonIterableSpread' in '/home/guo/git/ecom/laravel-react-demo/node_modules/@babel/runtime/helpers/esm'
Did you mean 'nonIterableSpread.js'?
BREAKING CHANGE: The request './nonIterableSpread' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

webpack compiled with 9 errors
```

### Steps To Reproduce:

```
git clone git@github.com:ecom/laravel-react-demo.git
cd laravel-react-demo
npm install
npm run hot
```

**Comment by guoyunhe** [comment]
It is a `@babel/runtime` fixed in `7.12.0` [issue](https://github.com/babel/babel/issues/12158). So a dependency version bump is needed.

**Comment by guoyunhe** [comment]
Solved by `rm -r package-lock.json node_modules && npm install`
